incubator/openebs/templates/cm-openebs-prometheus-config.yaml

```diff
@@ -0,0 +1,58 @@
+{{- if .Values.policies.monitoring.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "openebs.fullname" . }}-prometheus-config
+{{- if .Values.rbac.create }}
+  namespace: {{ .Values.rbac.namespace }}
```

Comment:
Why do you install resources into a different namespace depending on whether or not RBAC is enabled?

Reply:
Good point. This was used to enable/disable service account. I removed this check and kept it only for creating and associating service account.